### DISCUSSION TITLE
[Projects Beta] ETA on webhooks

### DISCUSSION THREAD

**Discussion by joshuaNathaniel** [community/community#8242] [Projects and Issues]
I can implement anything in GH Actions if I had the ability to trigger on beta project webhooks. Surely the default workflows use these triggers and they exist internally.

ETA?

**Comment by mikermcneil** [comment]
Would love to have this [in Fleet's boards](https://github.com/orgs/fleetdm/projects?type=beta) so that we could automatically re-set fields when moving issues across boards (i.e. from backlog/product drafting pipeline to engineering iteration board)

**Comment by naturallymitchell** [comment]
I think this is the issue to follow: https://github.com/github/roadmap/issues/483
